src/Pages/Cart/Cart.jsx

```diff
@@ -1,91 +1,141 @@
-import Navbar from "../../Component/Navbar/Navbar";
-import React from "react"
+import Navbar from '../../Component/Navbar/Navbar';
+import React from 'react';
+import { useCart } from '../../Context/Cart/Cart-Context';
 function Cart() {
-    return (
-      <>
-      <div className="h-100">
-  
+  const {
+    cart,
+    removeCart,
+    incrementQuantity,
+    decrementQuantity,
+    priceData,
+    moveToWishlist,
+  } = useCart();
+
+  const { totalDiscount, totalProductPrice, deliveryPrice } = priceData;
+  return (
+    <>
+      <div className='h-100'>
         <Navbar></Navbar>
-          <main className="wt-80 m-y-11 m-auto bg-black-0 flex flex-column items-center flex-wrap">
-  
-              <div className="text-center m-y-8">
-                  <h2 className="text-light text-center">My Carts</h2>
-              </div>
-              <div className="flex wt-100  justify-around pd-5 ">
-  
-                 
-                  <section className=" box-shadow flex justify-btw wt-50 m-b-8 rounded-s ">
-                      <img src="/css/image/strawberri.jpg" className="wt-50 pd-2 object-content m-r-6" alt=""/>
-  
-                      <div className="pd-11 wt-80 flex flex-column justify-even ">
-                          <div className=" flex flex-column items-start ">
-                              <div className="flex flex-column items-start ">
-                                  <span className="text-xm">Fruit</span>
-                                  <span className="text-bold text-xm">₹100</span>
-                                  <span className="text-bold">10%off </span>
-                              </div>
-                              <div className="flex justify-even items-center ">
-                                  <span>Quantity: </span>
-                                  <button className="cursor rounded-full wt-fixed-5 h-fixed-5 ">-</button>
-                                  <input type="number" placeholder="1" className="wt-fixed-9 h-fixed-6"/>
-                                  <button className="cursor rounded-full wt-fixed-5 h-fixed-5">+</button>
-                              </div>
-                          </div>
-  
-                          <div className="flex flex-column items-center m-y-3">
+        <main className='wt-80 m-y-11 m-auto bg-black-0 flex  items-center flex-wrap'>
+          <div className='text-center m-y-8'>
+            <h2 className='text-light text-center'>My Carts</h2>
+          </div>
+          <div className='flex wt-100 '>
+            <div className=' flex flex-column justify-around pd-5 '>
+              {cart.map((product) => {
+                const {
+                  _id,
+                  imageSrc,
+                  title,
+                  price,
+                  inStock,
+                  quantity,
+                  rating,
+                  qty,
+                  discount,
+                  categoryName,
+                } = product;
```

Comment:
i think you need to configure your fomatter extension

Reply:
prettier is working thats why is like that.